cartoonfight/player.py

```diff
@@ -78,7 +88,27 @@ def draw(self, sprites):
             pygame.draw.rect(self.display,(200,200,200),(window_size[0]*0.55,window_size[1]*0.05,window_size[0]*0.4,window_size[1]*0.03))
             pygame.draw.rect(self.display,color,(window_size[0]*0.55+window_size[0]*0.4*(1-player_health),window_size[1]*0.05,window_size[0]*0.4,window_size[1]*0.03))
 
+    #actions of the player
+    def mov_right(self):
```

Comment:
This move functions are fantastic! But  I think we should name it with its actual name: move_right